source/common/router/config_impl.cc

```diff
@@ -237,7 +237,9 @@ RouteEntryImplBase::RouteEntryImplBase(const VirtualHostImpl& vhost,
       response_headers_parser_(HeaderParser::configure(route.route().response_headers_to_add(),
                                                        route.route().response_headers_to_remove())),
       opaque_config_(parseOpaqueConfig(route)), decorator_(parseDecorator(route)),
-      direct_response_code_(ConfigUtility::parseDirectResponseCode(route)) {
+      direct_response_code_(ConfigUtility::parseDirectResponseCode(route)),
+      direct_response_body_(ConfigUtility::parseDirectResponseBody(route)),
+      direct_response_file_(ConfigUtility::parseDirectResponseFilename(route)) {
```

Comment:
I'd suggest checking for file existence and readability and throwing an error on failure.